modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -74,7 +80,7 @@ def phone_from_facility(facility)
 
       # captures area code (\d{3}) number (\d{3}-\d{4})
       # and optional extension (until the end of the string) (?:\sx(\d*))?$
-      phone_captures = phone.match(/^(\d{3})-(\d{3}-\d{4})(?:\sx(\d*))?$/)
+      phone_captures = phone.match(/^(\d{3})-?(\d{3}-?\d{4})(?:\sx(\d*))?$/)
```

Comment:
staging data had a phone number with no dashes so this should handle both cases.